pages/blog/product-metrics-open-source.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: "Measuring product usage for open source projects"
+date: "2020-12-22"
+author: "Andy Jih"
+pullQuote: "How we measure product and user metrics for our open source product"
+tags: [product]
+---
+
+A necessary practice and skill to building a successful product and business is having accurate, accessible, and actionable data. I’ve had the privilege and luck to work at some great companies that are extremely data-focused, and I’ve learned from some of the best along the way.
+
+**Quantitative user data**
+
+At [Zynga](http://zynga.com), we had millions of daily active users, where we tracked every click, action, and session across all of our games and apps. This data allowed us to conduct tons of multi-variate tests, often getting to statistical significance in a matter of hours. Having that wealth of data was both a blessing and a curse. From my experience, I was able to hone my skills at defining hypotheses, tests, analyzing the results quickly, and making decisions on features with confidence. The downside to this approach was that it became harder to push back on doing tests that would improve short-term metrics (revenue, feature usage), but harm long-term metrics (retention, NPS/CSAT).
+
+**Qualitative user data**
+
+During my time at [TaskRabbit](http://taskrabbit.com) and [Stripe](http://stripe.com), I really cut my teeth at conducting customer discovery and user research interviews. TaskRabbit and Stripe are both very different companies from Zynga, in that Zynga’s games are entertainment, while TaskRabbit’s and Stripe’s products are tools, providing utility to a person or business. While there are many product development principles that are applicable across many types of products, users of products that are tools often have a much clearer picture about the goals they’re trying to accomplish or the challenges they’re facing. Because of that difference, talking to users is often one of the best ways to learn very quickly where your product isn’t meeting expectations. I spent a ton of time working with my product, design, and research partners to define research goals, recruit users, conduct the interviews, and synthesize findings.
+
+## How open source metrics are different
+
+While we’re still in the early days at Grouparoo, I quickly realized I didn’t have as many of the tools at my disposal as I did in my previous roles. Unlike SaaS products where you have direct access to user and product usage data through SQL queries, we have to piece together metrics from a few different places. It's taken me a bit of time to adjust to this new landscape, so I thought I'd share our approach right now.
+
+**Discovery metrics**
+
+We can monitor discovery and adoption in a few different places. We have Google Analytics on our www site, as many people do, to get a rough sense for site visitors and traffic.
```

Comment:
Might not want to call out that we use GA... folks are pretty negative on them these days.  You could alternatively point to Cloudflare's analytics for the www site?  (screenshot in slack)